### ISSUE TITLE
Unable to create smoke by injecting people in cryo with sugar, phosphorous, and potassium

### ISSUE THREAD

**Issue by Cisrox** [d3athrow/vgstation13#17204]
#### Description of issue
i wanted to make a mutator, so I put a stasis beaker filled with smoke ingredients and unstable mutagen in a cryo cell, but when i turned it on no smoke was made

#### Difference between expected and actual behavior
expected smoke to be produced, no smoke was made

#### Steps to reproduce 
put man in cryo
put stasis beaker with smoke recipe in cryo
turn cryo on
no smoke produced

#### Specific information for locating
bodies and smoke

#### Length of time in which bug has been known to occur
not sure, but i used to be able to do this

#### Issue bingo

- [x] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [x] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

**Comment by FudgePucker** [comment]
Last time I tried to make a smoke reaction in cyro with stasis beakers it worked, this was about a year ago.

Also unstable mutagen smoke wouldn't work. The way cyro reactions work is by moving chemicals into the patent inside the tube, their internal storage isn't a stasis beaker so the reaction happens. Touch reaction regents, acid and mutagen to name two, will apply their touch effects to the patent without ever entering their internal storage. Smoke + touch reactions are pretty much the best use of smoke, to the point of being broken due to them applying the touch reactions thousands of times a second, so polyacid smoke is instakill. I recommend space lube smoke, it disables cyro until they manage to find a way to shut it down or the beaker runs dry.

**Comment by Cisrox** [comment]
yeah, i remember the smokes working a year ago because thats how i made the plasma smoke that destroyed the station, don't know why they don't work anymore

**Comment by Allspice** [comment]
Potentially caused by the interaction of #10060 probably preventing smoke from airtight things and #11829 probably making cryo airtight.

**Comment by FudgePucker** [comment]
Lame.
